meson_options.txt

```diff
@@ -14,6 +14,8 @@ option('docs', type: 'feature', value: 'auto',
     description: 'Build documentation')
 option('omf-byte-order', type: 'combo', choices: ['native', 'big', 'little'],
     value: 'native', description: 'On-media-format byte order')
+option('kvs-key-cpe', type: 'combo', choices: ['speed', 'size'],
```

Comment:
I would say drop the kvs prefix here. `key-cpe` sounds better.

Reply:
Yeah, originally I just had "cpe", then I added the extra stuff to future proof it.  I'll drop the kvs...

Reply:
Done.